nautobot/extras/forms/forms.py

```diff
@@ -605,10 +609,17 @@ class ExternalIntegrationBulkEditForm(NautobotBulkEditForm):
     verify_ssl = forms.NullBooleanField(required=False, label="Verify SSL", widget=BulkEditNullBooleanSelect)
     timeout = forms.IntegerField(required=False, min_value=0)
     extra_config = forms.JSONField(required=False)
+    http_method = forms.ChoiceField(
+        required=False,
+        label="HTTP Method",
+        choices=add_blank_choice(HTTPMethodClassChoices),
+    )
+    headers = forms.JSONField(required=False, label="HTTP Request headers")
+    ca_file_path = forms.CharField(required=False, label="CA File Path")
 
     class Meta:
         model = ExternalIntegration
-        nullable_fields = ["extra_config", "secrets_group"]
+        nullable_fields = ["extra_config", "secrets_group", "http_method", "headers", "ca_file_path"]
```

Comment:
Looking at the model, `http_method` and `ca_file_path` are not nullable, nor should they be.